File: 070.ClimbingStairs/wyrj/ClimbingStairs.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    var table = [1, 1], i;
    for (i = 2; i <= n; i++) {
        table[i] = table[i - 1] + table[i - 2];
    }
    return table[n];
};

File: 070.ClimbingStairs/Ddfish/ClimbingStairs.swift
func climbStairs(_ n: Int) -> Int {

	if (n <= 2) { return n }
	var stairs : [Int] = [0,1,2]

		for i in 3...n{
			stairs.append(stairs[i-1]+stairs[i-2])
		}

	return stairs[n]
}
